inference-engine/ie_bridges/python/src/openvino/inference_engine/ie_api.pyx

```diff
@@ -340,8 +340,6 @@ cdef class IECore:
         else:
             weights_ = "".encode()
             model = os.fspath(model)
-            if not os.path.isfile(model):
```

Comment:
I suggest changing this from the requirement for a path to be a file path to just checking the existence of the path with `os.path.exists()`